codeHF/PlotEfficiency_recoStep.C

```diff
@@ -0,0 +1,383 @@
+// Computation and plotting of reconstruction efficiency step-by-step
+// Four steps defined: kRecoHFFlag, kRecoTopol, kRecoCand, kRecoPID
+// Macros input: AnalysisResults.root from O2 with TH2F (pt vs y) distributions for generated and reconstructed candidates
+// .L PlotEfficiency_recoStep.C
```

Comment:
`PlotEfficiency_recoStep` -> `PlotEfficiencyRecoStep`.
Is there a reason why the main function does not have the same name as the file?

Reply:
Done

Reply:
The file is still called `PlotEfficiency_recoStep.C`.

Reply:
and the function is now called PlotEfficiencyRecoStep

Reply:
Yes, so the original comment has not been addressed.

Reply:
Done